Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -183,11 +203,17 @@ private bool FindClosestTouchableForLayerMask(LayerMask layerMask, out BaseNearI
                 Debug.LogWarning($"Maximum number of {numColliders} colliders found in PokePointer overlap query. Consider increasing the query buffer size in the input system settings.");
             }
 
+            Camera mainCam = CameraCache.Main;
             for (int i = 0; i < numColliders; ++i)
             {
-                var touchable = queryBuffer[i].GetComponent<BaseNearInteractionTouchable>();
+                var collider = queryBuffer[i];
+                var touchable = collider.GetComponent<BaseNearInteractionTouchable>();
                 if (touchable)
                 {
+                    if (ignoreCollidersNotInFOV && !mainCam.IsInFOVConeCached(collider))
```

Comment:
nit: Feel free to ignore this for now. But should use IgnoreCollidersNotInFOV public property instead of private, to be consistent with access to private variables in case the getter/setter does particular code check etcs

Reply:
Fixed.